kano_feedback/DataSender.py

```diff
@@ -61,7 +62,10 @@ def send_data(text, full_info, subject="", network_send=True):
     files = {}
     # packs all the information into 'files'
     if full_info:
-        files['report'] = get_metadata_archive()
+        if logs_path and os.path.exists(logs_path):
+            files['report'] = open(logs_path, 'rb')
```

Comment:
I really don't like how these are just files which are opened and handed off with somebody else responsible for closing them (or not in this case). At a minimum, the `get_metadata_archive()` should implement a context handler so you do a `with ...`, giving them a chance to properly clean up after going out of scope. Clearly not a problem introduced here but it is something that we should address moving forward.